src/WormholeJoin.sol

```diff
@@ -215,7 +224,7 @@ contract WormholeJoin {
     * @param batchedDaiToFlush Amount of DAI that is being processed for repayment
     **/
     function settle(bytes32 sourceDomain, uint256 batchedDaiToFlush) external {
-        require(batchedDaiToFlush <= 2 ** 255, "WormholeJoin/overflow");
+        if (batchedDaiToFlush > 2 ** 255) revert Overflow(batchedDaiToFlush);
```

Comment:
`NotAllowedBiggerInt256` and `Overflow` should be merged into `Overflow`.

Reply:
done in 0172295